### PR TITLE
Add default values for parameters for NI-DMM

### LINKED ISSUE #298: Add default parameter metadata to NI-DMM
Go through the API with a domain expert, and decide for each function which parameters should have default values in the Python API and what those values should be.

This needs to be done carefully, because in Python, all parameters with default values must go at the end of the parameter list. So changing this later would likely be a source breaker for client code.

This will most likely be a subset of Recommended/Optional input terminals on corresponding LabVIEW VIs.

### PR DESCRIPTION
**Pull request by Fladolcetta**
[ x] This contribution adheres to [CONTRIBUTING.md]

### What does this Pull Request accomplish?

Fixes #298 

### Why should this Pull Request be merged?

To add default metadata for NI-DMM

### What testing has been done?

Ran unit tests and system tests. Ran all examples on sample device.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,15 +12,19 @@ and this project adheres to [Python Versioning](http://legacy.python.org/dev/pep
 * #### NI-DMM
   * #### Added
   * #### Changed
+    * Added default values for some methods.
```
Comment (marcoskirsch) [inline CHANGELOG.md]: Added default values to some parameters.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,15 +12,19 @@ and this project adheres to [Python Versioning](http://legacy.python.org/dev/pep
...
   * #### Removed
+    * Removed some unnecessary methods.
```
Comment (marcoskirsch) [inline CHANGELOG.md]: Removed methods that aren’t useful in the Python bindings.

File: CHANGELOG.md (hunk 1 above)
Comment (Fladolcetta) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (Fladolcetta) [reply CHANGELOG.md]: Fixed

**Pull request merged**